content/docs/06-integrations/00-prometheus-operator.md

```diff
@@ -18,167 +18,601 @@ import Tooltip from "shared/components/ui/Tooltip";
+
+The Prometheus Operator pack has one parameter you must initialize `grafana.adminPassword`:
+
+<br />
+
+```yaml
+charts:
+  kube-prometheus-stack:
+    grafana:
+      adminPassword: ""
+```
+
+Use the `grafana.adminPassword` parameter to assign a password to the Grafana admin user `admin`. 
+
+Additional parameters you should be aware of can be found by expanding the **Presets** view of the pack. You can modify the preset settings during the profile creation process or the cluster deployment process when reviewing the cluster profile.
+![A view of the pack's preset drawer expanded with radio buttons](/integrations_prometheus-operator_operator-preset-view-expanded.png)
+
+Review the usage section below to learn more about each preset option.
+
+<br />
+
+## Usage
+
+Check out the guide [Deploy Monitoring Stack](/clusters/cluster-management/monitoring/deploy-monitor-stack) to learn how to create a monitoring stack with Prometheus for your Palette environment.
+
+<br />
+
+#### Email Alerts
+
+You can configure the Prometheus server to send email alerts to a set of contacts. Toggle the **Email Alerts** button to enable email alerting. Update the `alertmanager.config.receivers` settings with all the required email setting values.
+
+<br />
+
+```yaml
+charts:
+  kube-prometheus-stack:
+    alertmanager:
+      config:
+        receivers:
+          - name: email-alert
+            email_configs:
+              - to: <reciever>@<domainname>.com
+                send_resolved: true
+                from: <sender>@<domainname>.com
+                smarthost: smtp.<domainname>.com:587
+                auth_username: <sender>@<domainname>.com
+                auth_identity: <sender>@<domainname>.com
+                auth_password: <sender_passwd>
+```
+
+Refer to the [Prometheus Alertmanager Configuration](https://prometheus.io/docs/alerting/latest/configuration/) documentation to learn more about Alertmanager.
+
+<br />
+
+#### Grafana Ingress
+
+You can enable an ingress endpoint for Grafana that will deploy an NGINX ingress controller. This feature can be used to enable HTTPS and require authentication for all Prometheus API requests. 
+
+By default, a service with a load balancer, exposing port 80 is created unless the ingress option is enabled. 
+
+Toggle the **Enable** button to enable the use of Ingress.
+
+<br />
+
+
+#### Thanos SideCar
+
+[Thanos](https://prometheus-operator.dev/docs/operator/thanos/) is an open-source system for running large-scale, distributed, and highly available Prometheus setups. Thanos allows Prometheus to store data for extended periods by enabling data to be stored in object storage, such as Amazon S3 or Google Cloud Storage, instead of a local disk. This enables Prometheus to scale horizontally without needing to worry about running out of local storage space.

```

Comment:
```suggestion
[Thanos](https://prometheus-operator.dev/docs/operator/thanos/) is an open-source system for running large-scale, distributed, and highly available Prometheus setups. Thanos allows Prometheus to store data for extended periods in object storage, such as Amazon S3 or Google Cloud Storage, instead of a local disk. This enables Prometheus to scale horizontally without the risk of using up local storage space.

```